src/Authentication.Abstractions/AzureCmdletContext.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions.Interfaces;
+
+namespace Microsoft.Azure.Commands.Common.Authentication.Abstractions
+{
+    /// <summary>
+    /// Represents the context for an Azure cmdlet.
+    /// </summary>
+    public class AzureCmdletContext : ICmdletContext
+    {
+        private string cmdletId;
```

Comment:
```suggestion
        private string _cmdletId;
```
suggest using '_' as the prefix of private field.

Reply:
We have both private data members that start with lower case and "_". Let get a unified decision for it.